docusaurus/docs/advanced/fts.md

```diff
@@ -0,0 +1,205 @@
+    'index-hotel-description',
+    SearchQuery.conjuncts(
+      SearchQuery.match('five-star'),
+      SearchQuery.matchPhrase('luxury hotel')
+    )
+  )
+}
+
+var result = await ftsConjunction()
+console.log('RESULT:', result)
+````
+
+:::info
+Our match for `"five-star"` was not exact, but still produced a result because a similar term was found `"Five star"`, 
+we could have potentially matched `"5 star"` or the word `"five"`.
+When you work with any full-text search the number of hits you get and their score are variable.
+:::
+
+#### Disjunction
+
+A query satisfying (by default) one query or another. 
+If a conjunction query can be thought of like using an `AND` operator, a disjunction would be like using an `OR` operator.
+The example below will return seven documents hitting on the term `"Louvre"` and five hits on the term `"Eiffel"` returning a total of 12 rows together as part of a disjunction query.
+
+```javascript
+import { searchQuery, SearchQuery, TermSearchFacet } from 'ottoman';
+
+async function ftsDisjunction() {
+  return await searchQuery(
+    'index-hotel-description',
+    SearchQuery.disjuncts(
+      SearchQuery.match('Louvre'),
+      SearchQuery.match('Eiffel')
+    ),
+    {
+      facets: {
+        Descriptions: new TermSearchFacet('description', 5),
+      },
+      limit: 12,
+    }
+  )
+}
+
+result = await ftsDisjunction()
+console.log('RESULT:', result)
+```
+
+## Working with Results
+
+As with all query result types in Ottoman, the search query results object contains two properties. The hits reflecting the documents that matched your query, emitted as rows. Along with the metadata available in the meta property.
+
+Metadata holds additional information not directly related to your query, such as success total hits and how long the query took to execute in the cluster.
+
+#### Iterating over Hits
+
+```javascript
+result.rows.forEach((hit, index) => {
+  const docId = hit.id
+  const score = hit.score
+  const resultNum = index + 1
+  console.log(`Result #${resultNum} ID: ${docId} Score: ${score}`)
+})
+```
+
+#### Facets
+
+```javascript
+var facets = result.meta.facets
+console.log('Descriptions facet:', facets.Descriptions)
+```
+
+## Scan Consistency and ConsistentWith
+
+By default, all Search queries will return the data from whatever is in the index at the time of query. 
+These semantics can be tuned if needed so that the hits returned include the most recently performed mutations,
+at the cost of slightly higher latency since the index needs to be updated first.
+
+There are two ways to control consistency: either by supplying a custom `SearchScanConsistency` or using `consistentWith`.
+At the moment the cluster only supports `consistentWith`, which is why you only see `SearchScanConsistency.NotBounded`
+in the enum which is the default setting.
+The way to make sure that recently written documents show up in the search works as follows (commonly referred to "`read your own writes"` — RYOW):
```

Comment:
```suggestion
The way to make sure that recently written documents show up in the search works as follows (commonly referred to as **"read your own writes"**  or **RYOW**):
```

Reply:
Not really sure if this makes sense but wasn't sure if it should be a codeblock

Reply:
you're correct, the fix will be to remove the ` characters